### PR TITLE
Voting event handling added

### PR DESCRIPTION
**Pull request by hamalmar**

### CODE REVIEW

File: app/scripts/decisions.directive.js [hunk 1]
```diff
@@ -126,6 +126,37 @@ angular.module('dashboard')
...
+                if (angular.isObject(aVoting)) {
+                    $log.log("dbDecisions.updateVoting", arguments);
+                    if (!angular.isArray(self.voting)) {
+                        self.voting = [];
+                    }
+
+                    if (aVoting.typeName === CONST.MTGEVENT.VOTINGUPDATED || aVoting.typeName === CONST.MTGEVENT.VOTESINSERTED) {
+                        $log.log("dbDecisions.updateVoting,  mtgTopicSelected.topicGuid", mtgTopicSelected.topicGuid);
+                        if (angular.isObject(mtgTopicSelected) && angular.equals(mtgTopicSelected.topicGuid, aVoting.votings[0].topicGuid)) {
+                            if (angular.isObject(aVoting)) {
```
Comment (tkenakka) [inline app/scripts/decisions.directive.js:139]: Move isObject check to previous if, object property referenced already there

File: app/scripts/decisions.directive.js [hunk 2]
```diff
@@ -126,6 +126,37 @@ angular.module('dashboard')
...
+                                var found = false;
+                                for (var y = 0; y < aVoting.votings.length; y++) {
```
Comment (tkenakka) [inline app/scripts/decisions.directive.js:141]: All `aVoting.voting`items iterated here, but earlier only `voting[0].topicGuid` checked
Remove check or check all guids?

File: app/scripts/decisions.directive.js [hunk 3]
```diff
@@ -126,6 +126,37 @@ angular.module('dashboard')
...
+            function updateVoting(aVoting) {
+                if (angular.isObject(aVoting)) {
+                    $log.log("dbDecisions.updateVoting", arguments);
+                    if (!angular.isArray(self.voting)) {
+                        self.voting = [];
+                    }
+
+                    if (aVoting.typeName === CONST.MTGEVENT.VOTINGUPDATED || aVoting.typeName === CONST.MTGEVENT.VOTESINSERTED) {
+                        $log.log("dbDecisions.updateVoting,  mtgTopicSelected.topicGuid", mtgTopicSelected.topicGuid);
+                        if (angular.isObject(mtgTopicSelected) && angular.equals(mtgTopicSelected.topicGuid, aVoting.votings[0].topicGuid)) {
```
Comment (tkenakka) [inline app/scripts/decisions.directive.js:138]: I recommend to add an `else` with `log.error` for all cases where data is invalid and you should know about it whne bug hunting

**Comment by tkenakka** [comment]
LGTM if comments fixed

**Comment by hamalmar** [comment]
New pull request with fixes coming.

**Pull request closed without merging**